frontends/election-manager/src/components/hand_marked_paper_ballot.tsx

```diff
@@ -814,7 +814,7 @@ export function HandMarkedPaperBallot({
               </div>
             </PageFooterRow>
           </PageFooterMain>
-          {ballotMode === BallotMode.Sample ? (
+          {[BallotMode.Draft, BallotMode.Sample].includes(ballotMode) ? (
```

Comment:
As mentioned in the PR description, even though I later in this PR update system admin ballots to be sample ballots instead of draft ballots, draft ballots will make a return in the near future, so I'm not completely removing them from the code. (See the PR description for more detail.)

This change right here was just something that I missed when I first introduced draft ballots